group_vars/all.yml

```diff
@@ -2,9 +2,9 @@
 infrared_dir: "{{ ansible_user_dir }}/infrared"
 infrared_workspaces_dir: "{{ ansible_user_dir }}/.infrared/.workspaces"
 infrared_hosts_file: "{{ infrared_workspaces_dir }}/active/hosts"
-instackenv_file: "{{ ansible_user_dir }}/instackenv.json"
+instackenv_file: "{{ playbook_dir }}/instackenv.json"
```

Comment:
where the playbook_dir defined?

Reply:
@masco It's an ansible special var.
playbook_dir contains the playbook base directory.

Reply:
ack